### ISSUE TITLE
FEATURE: Prevent hidden files from being dimmed (darker color)

### ISSUE THREAD

**Issue by eric-freewill** [nvim-neo-tree/neo-tree.nvim#1493]
### Did you check the docs?

- [X] I have read all the docs.

### Is your feature request related to a problem? Please describe.

I've looked through the docs and code and I'm not seeing a way to disable hidden files from being dimmed or darker text. I work on hidden files often (github action workflows) and I can barely see the filenames and have tried a few different color schemes.

### Describe the solution you'd like.

This may possible already and I just missed it. It would be nice to turn off dimming of hidden files and have the highlight color be same as all other files.

**Comment by switch-blade-stuff** [comment]
I agree, having the ability to customize colors would be nice.
 
For now though you could either use `highlight! NeoTreeDotfile OtherHighlightGroup` or edit the [`highlights.lua`](https://github.com/nvim-neo-tree/neo-tree.nvim/blob/main/lua/neo-tree/ui/highlights.lua) file directly if I am not mistaken.

**Comment by ctrlplusb** [comment]
it'd also be rad to be able to set the highlight for hidden folder vs hidden file

**Comment by CX330Blake** [comment]
Agree, I want to customize the hidden file color but it didn't work.

**Comment by doongjohn** [comment]
> it'd also be rad to be able to set the highlight for hidden folder vs hidden file

oil.nvim has this feature: https://github.com/stevearc/oil.nvim/issues/578#issuecomment-2676305085

I really want this feature for neo-tree 😢
